cli/generative-ai/inference/stable_diffusion/text-to-image-online-endpoint.sh

```diff
@@ -0,0 +1,87 @@
+set -x
```

Comment:
Should this file be replaced with `cli/generative-ai/inference/cli/text-to-image-online-endpoint.sh`?